### PR TITLE
`azurerm_gallery_application_version` - Support `config_file_name`, `package_file_name` and `target_region.exclude_from_latest`

### PR DESCRIPTION
**Pull request by ms-zhenhua**
## Swagger Link:

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-03-03/gallery.json

## Test Results:
image

--- PASS: TestAccGalleryApplicationVersion_basic (868.44s)
--- PASS: TestAccGalleryApplicationVersion_requiresImport (877.59s)
--- PASS: TestAccGalleryApplicationVersion_complete (913.01s)
--- PASS: TestAccGalleryApplicationVersion_endOfLifeDate (1252.19s)
--- PASS: TestAccGalleryApplicationVersion_excludeFromLatest (1537.45s)
--- PASS: TestAccGalleryApplicationVersion_update (1539.47s)
--- PASS: TestAccGalleryApplicationVersion_enableHealthCheck (1576.67s)
--- PASS: TestAccGalleryApplicationVersion_tags (1669.11s)
--- PASS: TestAccGalleryApplicationVersion_targetRegion (1984.23s)

### CODE REVIEW

**Review by stephybun — COMMENTED** [review]
Thanks for this @ms-zhenhua. One minor comment on the property names otherwise this looks fine.

File: internal/services/compute/gallery_application_version_resource.go [hunk 1]
```diff
@@ -33,10 +33,12 @@ type GalleryApplicationVersionModel struct {
 	Name                 string            `tfschema:"name"`
 	GalleryApplicationId string            `tfschema:"gallery_application_id"`
 	Location             string            `tfschema:"location"`
+	ConfigFileName       string            `tfschema:"config_file_name"`
```
[the remark this answers is not in the document]
Comment (ms-zhenhua) [reply internal/services/compute/gallery_application_version_resource.go]: Updated. Thanks.

File: internal/services/compute/gallery_application_version_resource.go [hunk 2]
```diff
@@ -33,10 +33,12 @@ type GalleryApplicationVersionModel struct {
...
 	EnableHealthCheck    bool              `tfschema:"enable_health_check"`
 	EndOfLifeDate        string            `tfschema:"end_of_life_date"`
 	ExcludeFromLatest    bool              `tfschema:"exclude_from_latest"`
 	ManageAction         []ManageAction    `tfschema:"manage_action"`
+	PackageFileName      string            `tfschema:"package_file_name"`
```
[the remark this answers is not in the document]
Comment (ms-zhenhua) [reply internal/services/compute/gallery_application_version_resource.go]: Updated. Thanks.

**Comment by ms-zhenhua** [comment]
Hey @stephybun, thank you for your review. I have updated the code. Could you kindly take another review? Thanks.

## Test Results

--- PASS: TestAccGalleryApplicationVersion_requiresImport (918.56s)
--- PASS: TestAccGalleryApplicationVersion_basic (1001.28s)
--- PASS: TestAccGalleryApplicationVersion_complete (1004.58s)
--- PASS: TestAccGalleryApplicationVersion_endOfLifeDate (1137.79s)
--- PASS: TestAccGalleryApplicationVersion_update (1458.32s)
--- PASS: TestAccGalleryApplicationVersion_enableHealthCheck (1466.81s)
--- PASS: TestAccGalleryApplicationVersion_excludeFromLatest (1485.87s)
--- PASS: TestAccGalleryApplicationVersion_tags (1601.82s)

**Review by stephybun — APPROVED** [review]
Thanks @ms-zhenhua LGTM 👍

**Pull request merged**